### ISSUE TITLE
Even after removing 'volume:' from the display's config, volume is still set to 0

### ISSUE THREAD

**Issue by joelis10** [b0mbays/continuously_casting_dashboards#67]
As the title says, I've removed the 'volume:' part from the config for my display but on each cast, it still sets the volume very low. Only way to combat this is by setting the volume in the config to a higher one but even then it's still annoying.

**Comment by b0mbays** [comment]
Hi, it should default to 5 (out of 10) if it fails to retrieve the current volume from the device. This should happen after setting it to 0 for the cast (to stop the casting noise). Is this not happening? 

Could you post logs?

Thanks!

**Comment by joelis10** [comment]
after some more experience with it, it seems like it's only sometimes it doesn't get the volume and instead mutes it. where would I find the logs for when I see it do it again? i've looked in HA > Settings > System > Logs but can't find anything related to the integration there other than HA complaining about it being a custom component and not being officially tested

**Comment by joelis10** [comment]
yep, I see logs now but nothing indicating anything's gone wrong yet. I'll continue to monitor and try to reproduce it and if I can, I'll comment again with my logs.

**Comment by thewookiewon** [comment]
Hello, 

I've encountered the same issue although when when I hash out the volume line it defaults to level 1 volume.

I've not yet enacted Debug logging

**Comment by thewookiewon** [comment]
Seems to be adding a negative figure for volume '-1'

```
2024-06-18 09:24:06.900 DEBUG (MainThread) [custom_components.continuously_casting_dashboards.dashboard_caster] All device map: {'Google Nest Hub Max': {'instances': [{'dashboard_url': 'http://<REDACTED IP>:8123/givtcp-animated-dashboard/0?kiosk', 'dashboard_state_name': 'Dummy', 'media_state_name': 'PLAYING', 'volume': -1, 'start_time': datetime.time(7, 0), 'end_time': datetime.time(0, 0), 'instance_change': False, 'speaker_groups': None}], 'current_instance': 0}}
Current device map: {'Google Nest Hub Max': {'dashboard_url': 'http://<REDACTED IP>:8123/givtcp-animated-dashboard/0?kiosk', 'dashboard_state_name': 'Dummy', 'media_state_name': 'PLAYING', 'volume': -1, 'start_time': datetime.time(7, 0), 'end_time': datetime.time(0, 0), 'instance_change': False, 'speaker_groups': None}}
```

**Comment by goldrake1981** [comment]
Is there a way to leave the Nest Hub volume in the same state it was in before casting? when it transmits it makes that beep to change the volume which is very annoying

**Comment by joelis10** [comment]
apologies for not finding the logs yet but it still happens on all my casted displays, even when omitting the `volume:` line
